compiler/rustc_session/src/config.rs

```diff
@@ -1762,6 +1762,10 @@ pub fn build_session_options(matches: &getopts::Matches) -> Options {
         debugging_opts.symbol_mangling_version = SymbolManglingVersion::V0;
     }
 
+    if let Some(graphviz_font) = option_env!("RUSTC_GRAPHVIZ_FONT") {
```

Comment:
This inspects the variable at compile-time. You want `std::env::var{_os}`.

Reply:
Oh, right. OK.

(I wanted to push this before waiting for my test to compile.)

But generally you're OK with the env var name, etc.? 

I'll fix how it's evaluated.

Reply:
Assuming it works, everything else looks good to me!

Reply:
Done. I waited for it to compile this time ;-) ... tested locally with the environment variable set and unset and it works.